src/api/java/baritone/api/utils/BlockOptionalMeta.java

```diff
@@ -315,4 +315,8 @@ public IBlockState getAnyBlockState() {
 
         return null;
     }
+
+    public Set<IBlockState> getAllBlockStates() {
+        return blockstates;
```

Comment:
Is this set immutable? If not it should be made immutable or wrapped in an unmodifiable set.